curriculum/challenges/english/01-responsive-web-design/basic-html-and-html5/define-the-head-and-body-of-an-html-document.english.md

```diff
@@ -46,10 +46,9 @@ tests:
   - text: The <code>body</code> element should be a child of the <code>html</code> element.
     testString: assert($('html').children('body').length == 1);
   - text: The <code>head</code> element should wrap around the <code>title</code> element.
-    testString: assert(code.match(/<head>\s*?<title>\s*?.*?\s*?<\/title>\s*?<\/head>/gi));
+    testString: assert($('head').children('title').length == 1);
```

Comment:
This test is failing

Reply:
But the logic is correct. I don't know how the test files are being converted. I guess it has something to do with that. I am using the same JQuery concept that the upper test cases have used. 
See issue #39111

Reply:
Yup. See below